editor/editor_properties_array_dict.cpp

```diff
@@ -240,6 +234,48 @@ void EditorPropertyArray::_object_id_selected(const StringName &p_property, Obje
 	emit_signal(SNAME("object_id_selected"), p_property, p_id);
 }
 
+void EditorPropertyArray::create_new_property_slot() {
+	int idx = slots.size();
+	HBoxContainer *hbox = memnew(HBoxContainer);
+
+	Button *reorder_button = memnew(Button);
+	reorder_button->set_icon(get_editor_theme_icon(SNAME("TripleBar")));
+	reorder_button->set_default_cursor_shape(Control::CURSOR_MOVE);
+	reorder_button->set_disabled(is_read_only());
+	reorder_button->connect("gui_input", callable_mp(this, &EditorPropertyArray::_reorder_button_gui_input));
+	reorder_button->connect("button_up", callable_mp(this, &EditorPropertyArray::_reorder_button_up));
+	hbox->add_child(reorder_button);
+	EditorProperty *prop = memnew(EditorPropertyNil);
+	hbox->add_child(prop);
+
+	reorder_button->connect("button_down", callable_mp(this, &EditorPropertyArray::_reorder_button_down).bind(idx));
```

Comment:
Why isn't this with the other connections?

Reply:
no reasons. Moved it